script.js

```diff
@@ -1,4 +1,435 @@
+  }
+
+  // IF condition to take into account last winner in order to determine the ultimate winner
+  if (draw) {
+    if (lastWinner == 2) {
+      myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>You played ${input} ${iconPlayer}.<br>It's a draw.<br>As computer won the previous game, computer is the Ultimate Winner!`;
+    }
+    if (lastWinner == 1) {
+      myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>You played ${input} ${iconPlayer}.<br>It's a draw.<br> As ${userName} won the previous game, ${userName} is the Ultimate Winner!`;
+    }
+    // account for situation of first game draw and there is no previous winner
+    if (lastWinner == 0) {
+      myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>You played ${input} ${iconPlayer}.<br>It's a draw.<br> There is no previous winner, please play again.`;
+    }
+  }
+  console.log("lastWinner", lastWinner);
+  return myOutputValue;
+};
+
+// ---------------- Computer vs Computer (AI) mode ---------
+// ---------------------------------------------------------
+
+// notes will only show the difference from normal game mode
+
+var aiGame = function () {
+  var computerMove = randomItem();
+  console.log("computer", computerMove);
+
+  var iconComputer = iconLogo(computerMove);
+  console.log("iconComputer", iconComputer);
+
+  // add var for computer to select for player
+  var playerMove = randomItem();
+
+  // there is no need to into account players "input"
+  var iconPlayer = iconLogo(playerMove);
+  console.log("iconPlayer", iconPlayer);
+
+  var draw = drawGame(playerMove, computerMove);
+  var win = playerWin(playerMove, computerMove);
+
+  numGamesPlayed += 1;
+  console.log("numGamesPlayed", numGamesPlayed);
+
+  // changed all the text to better describe the game
+
+  var myOutputValue = `The computer played ${computerMove} ${iconComputer} .<br>AI played for you ${playerMove} ${iconPlayer}.<br>You lost ='( <br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+
+  if (win) {
+    numGamesWon += 1;
+    myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>AI played for you ${playerMove} ${iconPlayer}.<br>You won!!!<br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+  }
+
+  if (draw) {
+    numGamesDraw += 1;
+    myOutputValue = `The computer played ${computerMove} ${iconComputer}.<br>AI played for you ${playerMove}.<br>It's a draw!<br>${userName}, you have played ${numGamesPlayed} games and your win percentage is ${percentageWin()}%.<br>Lets play another round!`;
+  }
+  console.log("numGamesPlayed after ifs", numGamesPlayed);
+  console.log("numGamesWon after ifs", numGamesWon);
+  return myOutputValue;
+};
```

Comment:
Because of the way you've thought through and simplified your code from earlier version, you notice that you can easily replicate to different versions. 

So well done on that note 💪